### ISSUE TITLE
Add a config variable for time to wait between invites

### ISSUE THREAD

**Issue by BenignBeppe** [Wikimedia-Sverige/hostbot#21]

**Comment by BenignBeppe** [comment]
This can be done something like this (in "main" in send_th_invites):
```
for invitee in invitees:
    t = time.clock()
    DO INVITE STUFF
    d = time.clock() - t
    time.sleep(max(0, 10 - d))
```

**Comment by lokal-profil** [comment]
I would stick the 10s in the config (different limits for different wikis) but otherwise the solution looks good.

**Comment by BenignBeppe** [comment]
This is also added for tomorrows testing, along with #22.

**Comment by BenignBeppe** [comment]
Branch _throttle-api-requests_ is ready to go, but I'll wait to make the pull request until #25 is done, since it depends on _variable-config-file_.

**Comment by BenignBeppe** [comment]
Closed by ab6f9f665e008f5cc5daba547832c3aa58cbc750. Closing manually since I messed up the merge message.
